orderbook/usecase/orderbook_usecase_test.go

```diff
@@ -35,6 +35,101 @@ func TestOrderbookUsecaseTestSuite(t *testing.T) {
 	suite.Run(t, new(OrderbookUsecaseTestSuite))
 }
 
+// Creates a new osmomath.Dec from a string
+// Panics if the string is invalid
+func (s *OrderbookUsecaseTestSuite) newDecFromStr(str string) osmomath.Dec {
+	s.T().Helper()
+	dec, err := osmomath.NewDecFromStr(str)
+	s.Require().NoError(err)
+	return dec
+}
+
```

Comment:
Consider directly using and removing the helper:
```go
osmomath.MustNewDecFromStr(...)
```

Reply:
Updated accordingly :+1: